lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,9 +8,13 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
+          attr_accessor :#{column}_data_uri_cache
           attr_accessor :#{column}_data_filename, :#{column}_data_mimetype
 
           def #{column}_data_uri=(data)
+            return if data.blank?
```

Comment:
If we leave it like this here, then `Parser` won't be raising an error on empty data anymore and this can break existing integrations. We need to remove this line